### ISSUE TITLE
Missing "alt" release event, OR Why modifier key state should be part of keyboard input event

### ISSUE THREAD

**Issue by tomaka** [tomaka/winit#77]
Original: https://github.com/tomaka/glutin/issues/802

**Comment by rigtorp** [comment]
@jwilm I checked the Alt-Tab issue on Centos 7 and it doesn't exist for this version of winit: https://github.com/tomaka/winit/commit/09e6698236bab971c2903df96e596ad770a94b2e

Alt-Tab out from window:
```
KeyboardInput(Pressed, 64, Some(LAlt), )
Focused(false)
MouseLeft
Focused(false)
```

Alt-Tab in to window:
```
Focused(true)
MouseEntered
Focused(true)
```

So it would be possible to track these states.

**Comment by jwilm** [comment]
I'm not sure what you mean by "[the issue] doesn't exist". You did not get a release event for the LAlt key at any point in those logs.

**Comment by rigtorp** [comment]
When the window is focused you are guaranteed that Alt is released.

Another problem is if you hold down alt and then focus the window with your
mouse. The window never receives any Alt keypress in that case.

**Comment by francesca64** [comment]
This issue was resolved by PR #112
